library/src/morpheus-master/sass/modules/tool/conversations/_sakai-add-topic.scss

```diff
@@ -97,10 +97,6 @@
     margin-top: 10px;
   }
 
-  .active {
-    background-color: #A0D3F2;
-  }
-
```

Comment:
Not sure about removing this, I thought this was used to identify the currently active topic/posts.
If we keep it, I'd suggest
```
.active {
background-color: var(--sakai-active-color-1);
}
```

Reply:
I don't know where that .active refers to... but it's a bad CSS rule because is affecting all active buttons.

![image](https://user-images.githubusercontent.com/15141743/150496021-f3aa39cd-192a-4962-a31d-0e63b2b0495a.png)

Better removing that including a bad CSS rule.

Reply:
I thought namespacing it on line 1 of this file would resolve that issue. If not, then I agree.

Reply:
@profmikegreene If I'm not wrong, @adrianfish changed that class on conversations so it's not used anymore.